src/main/java/org/mule/extension/runtime/OperationContext.java

```diff
@@ -33,4 +34,14 @@
      * mean that the parameter is not present. It might have just been resolved to that value
      */
     Object getParameterValue(String parameterName);
+
+    /**
+     * Returns an object which is configuring the operation being executed. The actual type
+     * of the instance is unknown, but it's guaranteed to be a realisation of the {@link Configuration}
+     * model that was set for the operation
+     *
+     * @param <C> the generic type of the configuration instance
+     * @return an {@link Object} consistent with a corresponding {@link Configuration} model
```

Comment:
Javadoc says that "returns an Object", but it returns "C", fix javadoc or method definition